mobify/navitron
# Release v1.0.0

## Changes
- Initial v1.0.0 NPM release
- Use jQuery and latest Velocity release
- Migrate from Bower to NPM for dependencies
- Migrate JS linting to eslint
- Migrate SCSS compilation to libsass
- Lint SCSS with sass-lint
- Update circleCI config with modern testing steps
- Add some GitHub templates for contributing, issues, and PRs
- Fix circleCI tests (phantomJS does not support requestAnimationFrame)
## TODOs:
- [x] Tests pass locally (`grunt test:browser`)
- [x] Tests pass on CircleCI
- [x] Post Merge: Publish on NPM :shipit: 
## How To Test
- Checkout code
- Refresh node modules with `rm -rf node_modules && npm install`
- Run tests locally (`grunt test:browser`)
- Check that the examples work (`grunt examples`)
- Ensure tests pass on CircleCI
